user_tools/src/spark_rapids_pytools/build.py

```diff
@@ -27,3 +27,21 @@ def get_version(main=None):
     if nightly == '1':
         suffix = '.dev' + datetime.datetime.utcnow().strftime('%Y%m%d%H%M%S')
     return main + suffix
+
+
+def get_runtime_buildver(buildver_arg: str = None) -> str:
+    """
+    Get the runtime SPARK build_version for the user tools environment.
+    Note that the env_var always have precedence over the input argument and the default values
+    :param buildver_arg: optional argument to specify the build version
+    :return: the first value set in the following order:
+       1- env_var RAPIDS_USER_TOOLS_RUNTIME_BUILDVER
```

Comment:
can we put SPARK in the name of this so its more obvious that its the Spark version.  And actually BUILDVER here, are we building with it or are we running with it?  is there issues if those don't match (ie what we build with versus run with)?

I would rather just see this called like SPARK_RUNTIME_VERSION or SPARK_DEP_VERSION

Reply:
The user-tools used env_var with pattern `RAPIDS_USER_TOOLS_*`. The reason for that was to prevent overlap with other env_vars and allow user to specify env_vars without overriding the global environment. It is more consistent to have the same pattern for newly introduced env_vars
examples env_vars
`RAPIDS_USER_TOOLS_CACHE_FOLDER`
`RAPIDS_USER_TOOLS_OUTPUT_DIRECTORY`
`RAPIDS_USER_TOOLS_DATABRICKS_PROFILE`

Reply:
Renamed to 1RAPIDS_USER_TOOLS_SPARK_DEP_VERSION1